blueocean-personalization/package.json

```diff
@@ -50,33 +51,27 @@
     "@jenkins-cd/design-language": "0.0.158",
     "@jenkins-cd/js-extensions": "0.0.42",
     "@jenkins-cd/js-modules": "0.0.10",
-    "immutable": "3.8.1",
-    "keymirror": "0.1.1",
-    "react-i18next": "1.10.1",
-    "react-redux": "4.4.5",
+    "mobx": "2.6.0",
+    "mobx-react": "3.5.7",
     "react": "15.4.2",
     "react-addons-css-transition-group": "15.4.2",
     "react-dom": "15.4.2",
-    "react-router": "2.8.1",
-    "redux": "3.6.0",
-    "redux-thunk": "2.1.0",
-    "reselect": "2.5.4"
+    "react-i18next": "1.10.1",
```

Comment:
is `react-i18next` actually required? I figured that would come in transitively from core?

Reply:
might be ok to explicitly include it?